### PR TITLE
Change main layout to use application layout from react-components.

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
This is ready for review now

File: ui/src/components/Navigation.tsx [hunk 1]
```diff
@@ -13,125 +12,76 @@ type Props = {
...
-                      Logout
-                    </Button>
-                  </li>
-                </ul>
-              </div>
-            </div>
-          </div>
-        </div>
-      </nav>
+      <SideNavigation<NavLinkProps>
```
Comment (Lukewh) [inline ui/src/components/Navigation.tsx:15]: I've never seen this before, is this legit?

Comment (huwshimi) [reply ui/src/components/Navigation.tsx:15]: Hah yeah it's how you pass a TypeScript generic to a React component.

Comment (Lukewh) [reply ui/src/components/Navigation.tsx:15]: oooh of course it's a type 🤦! I thought it was a JSX Component within a JSX Component definition... THIS makes more sense!

**Review by Lukewh — APPROVED** [review]
:+1: LGTM - types in JSX components and all.

**Review by BarcoMasile — APPROVED** [review]

**Pull request merged**